mdf_toolbox/utils.py

```diff
@@ -0,0 +1,27 @@
+"""Utility functions
+
+These functions are intended for internal use by mdf_toolbox
+"""
+from pathlib import PureWindowsPath
+import re
+
+
+def rectify_path(path: str) -> str:
+    """Ensure that a path is ready for use with Globus.
+
+    Globus requires that paths are in POSIX format.
+    Windows paths should be converted to POSIX style,
+    where the "Drive" is listed as the
+    first folder (e.g., ``/c/Users/globus_user/``).
+
+    Args:
+        path (str): Input path
+    Returns:
+        (str) Rectified path
+    """
+
+    is_windows = re.match('[A-Z]:\\\\', path) is not None
+    if is_windows:
+        ppath = PureWindowsPath(path)
+        return f'/{ppath.drive[:1].lower()}{ppath.as_posix()[2:]}'
```

Comment:
Python's f-strings were introduced in 3.6, but the rest of Toolbox works on previous versions. Can this be changed to `.format()` instead, to maintain compatibility?

Reply:
Sure, I can swap it over to ".format"